pkg/controllers/provisioning/provisioner.go

```diff
@@ -176,14 +177,14 @@ func (p *Provisioner) consolidationWarnings(ctx context.Context, po *v1.Pod) {
 	if po.Spec.Affinity != nil && po.Spec.Affinity.PodAntiAffinity != nil {
 		if len(po.Spec.Affinity.PodAntiAffinity.PreferredDuringSchedulingIgnoredDuringExecution) != 0 {
 			if p.cm.HasChanged(string(po.UID), "pod-antiaffinity") {
-				logging.FromContext(ctx).Infof("pod %q has a preferred Anti-Affinity which can prevent consolidation", client.ObjectKeyFromObject(po))
+				log.FromContext(ctx).Info(fmt.Sprintf("pod %s has a preferred Anti-Affinity which can prevent consolidation", client.ObjectKeyFromObject(po)))
```

Comment:
If we find ourselves doing this, it strikes me that we should just make this a structured value and use `WithValues`

Reply:
```go
log.FromContext(ctx).WithValues("pod", client.ObjectKeyFromObject(po)).V(0).Info("has a preferred Anti-Affinity which can prevent consolidation")
```
@jonathan-innis would something like this be ok?

Reply:
I to be frank don't understand what would be the real benefit.

Reply:
It's the difference between structured values for a log parser vs. something having to parse out the pod from the message. It also carries a stronger contract than the message ordering. The way we word a message could be easily broken, but a structured value is much easy to keep consistent and ensure. In either case, we can take this as a follow-on since it's keeping the same structure and is out of scope of the PR.